content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,101 @@ content_template: templates/concept
-* Have a dedicated SRE team
-* Have the resources to host and monitor your clusters
-
-Pick an [on-prem turnkey cloud solution](/docs/setup/pick-right-solution/#on-premises-turnkey-cloud-solutions).
-
-## Custom Solutions
-
-Custom solutions give you the most freedom over your clusters but require the 
-most expertise. These solutions range from bare-metal to cloud providers on 
-different operating systems.
-
-Pick a [custom solution](/docs/setup/pick-right-solution/#custom-solutions).
+## Learning environment
+
+If you want to get familiar with Kubernetes, you can use the Docker-based solutions. You can use tools that are supported by the Kubernetes community or the tools in the ecosystem to setup a Kubernetes cluster on a local-machine. 
+
+{{< table caption="Local-machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
+
+|Community           |Ecosystem     |
+| ------------       | --------     |
+| [Minikube](/docs/setup/minikube/) | [CDK on LXD](https://www.ubuntu.com/kubernetes/docs/install-local) |
+| [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) | [Docker Desktop](https://www.docker.com/products/docker-desktop)|
+| [Kubernetes IN Docker](https://github.com/kubernetes-sigs/kind) | [Minishift](https://docs.okd.io/latest/minishift/)|
+|                     | [MicroK8s](https://microk8s.io/)|
+|                     | [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) |
+|                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
+|                     | [k3s](https://k3s.io)|
+|                     | [Ubuntu on LXD](/docs/getting-started-guides/ubuntu/)|
+
+
+## Production environment
+
+The production environment solutions diagram lists the possible abstractions of a Kubernetes cluster and the level of expertise involved in managing the operations of a cluster.
+
+Production environment solutions![Production environment solutions](/images/docs/KubernetesSolutions.svg)
+
+
+ 
+### Applications
+The layer where various containerized applications run. 
+ 
+### Data Plane
+The layer that provides capacity such as CPU, memory, network, and storage so that the containers can run and connect to a network.
+
+### Control Plane
+The container orchestration layer that exposes the API and interfaces to define, deploy, and manage the lifecycle of containers. 
+
+### Cluster Infrastructure
+The infrastructure layer provides and maintains VMs, networking, security groups and others. 
+
+### Cluster Operations
+Activities such as upgrading the clusters, implementing security, storage, ingress, networking, logging and monitoring, and other operations involved in managing a Kubernetes cluster.
+
+{{< table caption="Production environment solutions table that lists the solutions and the providers." >}}
```

Comment:
I think this table needs some introductory text.

Reply:
added an introduction.